src/client/lib/utils.js

```diff
@@ -125,14 +125,13 @@ export var createElement = function(type, attrs={}, text=null) {
 };
 
 
-
-export var toArray = (arrayLike) => {
+export let toArray = (arrayLike) => {
   var arr = [];
   try {
     arr = [].slice.call(arrayLike);
-  } catch(e) {
+  } catch (e) {
     // Thanks a lot IE 8. I mean it.
-    for (var i = 0; i < arrayLike.length; i++) {
+    for (let i = 0; i < arrayLike.length; i++) {
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).